src/main/java/com/galbur/methodstask/Customer.java

```diff
@@ -0,0 +1,57 @@
+package com.galbur.methodstask;
+
+public class Customer {
+    private String name;
+    private int age;
+    private String email;
+
+    public Customer(String name, int age, String email) {
+        this.name = name;
+        this.email = email;
+        if (age <= 1 || age >= 127) {
```

Comment:
Age can be 1, I would set the < operator instead

Reply:
did that from the start, then during watching the video applied <=,  but definitely worth changing it. thank you for your review